dynamicEDT3D/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 PROJECT(dynamicEDT3D)
 
 include(CTest)
+include(GNUInstallDirs)
```

Comment:
From a quick look, this probably requires a higher CMAKE_MINIMUM_REQUIRED version, is that correct?

Reply:
After a bit of archaeology, it seems that [cmake 2.8.8 does have that module](https://gitlab.kitware.com/cmake/cmake/blob/v2.8.8/Modules/GNUInstallDirs.cmake).